test/test-applications/integrations/Samples.WebRequest/Properties/launchSettings.json

```diff
@@ -12,7 +12,8 @@
           "CORECLR_PROFILER_PATH": "$(ProjectDir)$(OutputPath)profiler-lib\\OpenTelemetry.AutoInstrumentation.ClrProfiler.Native.dll",
   
           "OTEL_DOTNET_TRACER_HOME": "$(ProjectDir)$(OutputPath)profiler-lib",
-          "OTEL_INTEGRATIONS": "$(ProjectDir)$(OutputPath)profiler-lib\\integrations.json"
+          "OTEL_INTEGRATIONS": "$(ProjectDir)$(OutputPath)profiler-lib\\integrations.json",
+          "OTEL_PROPAGATORS": "datadog;b3"
```

Comment:
The issue might be with the semicolon as the separator. Currently, `StringConfigurationSource.ParseCustomKeyValues` only uses a `,` as a separator

Reply:
ConfigurationSourceExtensions defines ';' as Separator. This is currently dealing with string -> enum list conversion.

Reply:
@RassK This is wrong. It should be `,`. See the [specs](https://github.com/open-telemetry/opentelemetry-specification/blob/3f07b6669dbbf4625579b5a0f371768f50788cd2/specification/sdk-environment-variables.md#general-sdk-configuration).

Reply:
@pellared Seems you are right, it's a spec bug. `;` was already previously existing for other stuff (See https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/104/files#diff-5c417e25757186c840db429ba030b837d74ca21b2fcfdb9a06f9b96cfc24187eL62). We could overwrite the separator only for `OTEL_PROPAGATORS`.

Reply:
Personally, I would prefer to use `,` as the only separator.
Would changing the separator here: https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/104/files#diff-5c417e25757186c840db429ba030b837d74ca21b2fcfdb9a06f9b96cfc24187eL62 break anything?
Does anyone have any concerns?

Reply:
@zacharycmontoya any feedback if we are ok to change it to `,` ?

Reply:
Yeah changing it to `,` sounds good to me

Reply:
Seems some of the environment variables are shared between native level. For eg the same `OTEL_DISABLED_INTEGRATIONS` has `;` defined in util.cpp (GetEnvironmentValues method). Changing `;` to `,` in util.cpp would probably affect another set of variables. So it seems to be entire new PR...

Reply:
@RassK I think putting it in a new PR is a great idea.